.scripts/process_build.sh

```diff
@@ -0,0 +1,175 @@
+#!/usr/bin/env bash
+
+# Sorry, everyone
+if (( ${BASH_VERSION%%.*} < 4 )); then
+  echo "This script requires Bash >= 4. On OSX, try: brew install bash"
+  exit 1
+fi
+
+# Any error is fatal.
+set -e
+set -o pipefail
+
+BUILD_DIR=$(pwd)
+if [[ -f ~/.bash_profile ]]; then
+  source ~/.bash_profile
+fi
+cd "$BUILD_DIR"
+
+# Constants for showing color in output window
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+NC='\033[0m' # No Color
+
+CURRENT_VERSION=$(head -n 1 version.txt)
+
+echo "*** Clean up log files ***"
+if [[ -f src/ocaml-output/fstar/MicroBenchMarkOutput.log ]]; then
+  rm src/ocaml-output/fstar/MicroBenchMarkOutput.log
+fi
+if [[ -f src/ocaml-output/fstar/HelloOcamlOutput.log ]]; then
+  rm src/ocaml-output/fstar/HelloOcamlOutput.log
+fi
+if [[ -f src/ocaml-output/fstar/HelloFStarOutput.log ]]; then
+  rm src/ocaml-output/fstar/HelloFStarOutput.log
+fi
+if [[ -f src/ocaml-output/fstar/AllExamples.log ]]; then
+  rm src/ocaml-output/fstar/AllExamples.log
+fi
+
+# Need this to get back after unzip things
+ORIG_PWD=$PWD
+
+echo "*** Make package ***"
+cd src/ocaml-output
+make package
+
+# 'make package' makes the package using the major version from version.txt. This script is a weekly process to make minor versions so use timestamp in file name instead of major version
+echo "*** Unzip and verify the Package  ***"
+TIME_STAMP=$(date +%s)
+
+TYPE="_Windows_x64.zip"
+MAJOR_ZIP_FILE=fstar_$CURRENT_VERSION$TYPE
+if [[ -f $MAJOR_ZIP_FILE ]]; then
+  MINOR_ZIP_FILE=fstar_$TIME_STAMP$TYPE
+  cp $MAJOR_ZIP_FILE $MINOR_ZIP_FILE
+  unzip -o $MAJOR_ZIP_FILE
+else
+  TYPE="_Linux_x86_64.tar.gz"
+  MAJOR_TAR_FILE=fstar_$CURRENT_VERSION$TYPE
+  if [[ -f $MAJOR_TAR_FILE ]]; then
+    MINOR_TAR_FILE=fstar_$TIME_STAMP$TYPE
+    cp $MAJOR_TAR_FILE $MINOR_TAR_FILE
+    tar -x -f $MAJOR_TAR_FILE
+  else
+    echo -e "* ${RED}FAIL!${NC} src/ocaml-output/make package did not create "$MAJOR_ZIP_FILE" or "$MAJOR_TAR_FILE
```

Comment:
Super minor: you may want to use a consistent quoting style for variables within double-quoted strings, e.g.

```
echo -e "* ... did not create ${MAJOR_ZIP_FILE} or ${MAJOR_TAR_FILE}"
```